inputs/LiMetal_PorousSep_oxygen.yaml

```diff
@@ -0,0 +1,358 @@
+    first-step: 'discharge'  # Start with charge or discharge?
+    phi-cutoff-lower: 2.2 # Simulation cuts off if E_Cell <= this value
+    phi-cutoff-upper: 6.5 # Simulation cuts off if E_Cell >= this value
+    species-cutoff: 1e-12 # Simulation cuts off if C_k <= this value
+
+# Cantera inputs:
+description: |-
+  Cantera input file for an LCO/graphite lithium-ion battery
+
+  This file includes a full set of thermodynamic and kinetic parameters of a
+  lithium-ion battery, in particular:
+  - Active materials: LiCoO2 (LCO) and Li (li metal)
+  - Organic electrolyte: EC/PC with 1M LiPF6
+  - Interfaces: LCO/electrolyte and Li/electrolyte
+  - Charge-transfer reactions at the two interfaces
+
+  Reference:
+  M. Mayur, S. C. DeCaluwe, B. L. Kee, W. G. Bessler, “Modeling and simulation
+  of the thermodynamics of lithium-ion battery intercalation materials in the
+  open-source software Cantera,” Electrochim. Acta 323, 134797 (2019),
+  https://doi.org/10.1016/j.electacta.2019.134797
+
+  Bulk phases
+  ===========
+
+  Lithium (anode)
+
+  Lithium cobalt oxide (cathode)
+  Thermodynamic data based on half-cell measurements by K. Kumaresan et al.,
+  J. Electrochem. Soc. 155, A164-A171 (2008)
+
+  Carbonate based electrolyte (electrolyte)
+  Solvent: Ethylene carbonate:Propylene carbonate (1:1 v/v)
+  Salt: 1M LiPF6
+
+  Interface phases
+  ================
+
+  lithium/electrolyte interface (lithium_electrolyte)
+  Species and site density are dummy entries (as we do not consider surface-
+  adsorbed species)
+
+  LCO/electrolyte interface (cathode_electrolyte)
+  Species and site density are dummy entries (as we do not consider surface-
+  adsorbed species)
+
+generator: cti2yaml
+cantera-version: 2.5.0
+date: Wed, 11 Dec 2019 16:59:07 -0500
+input-files: [lithium_ion_battery.cti]
+
+phases:
+
+- name: lithium_metal
+  thermo: ideal-condensed
+  species: ['Li[metal]']
+  density: 0.534 g/cm^3
+  state:
+    T: 300.0
+    P: 1 atm
+- name: cathode
+  thermo: ideal-condensed
+  elements: [Li, C, O, Mn]
+  species: ['C[cathode]', 'Li2O2[cathode]', 'Cat[cathode]']
+  state:
+    X: {'C[cathode]': 0.80, 'Li2O2[cathode]': 0.00, 'Cat[cathode]': 0.20 }
+  standard-concentration-basis: unity
+  tabulated-species: Li2O2[cathode]
+- name: electrolyte
+  thermo: ideal-condensed
+  elements: [Li, P, F, C, H, O, E]
+  species: ['C3H4O3[elyt]', 'Li+[elyt]', 'PF6-[elyt]', O2(e), LiO2]
+  state:
+    X: {'C3H4O3[elyt]': 0.85464, 'Li+[elyt]': 0.07268,
+      'PF6-[elyt]': 0.07268}
+  standard-concentration-basis: unity
+  reactions: [electrolyte-reactions]
+- name: electron
+  thermo: electron-cloud
+  elements: [E]
+  species: [electron]
+  state:
+    X: {electron: 1.0}
+  density: 1.0 kg/m^3
+- name: lithium_electrolyte
+  thermo: ideal-surface
+  species: [(dummy)]
+  kinetics: surface
+  reactions: [lithium_electrolyte-reactions]
+  site-density: 0.01 mol/cm^2
+- name: air-elyte-interface
```

Comment:
Just realized that this phase is never imported in your `initialize` routine.  Let's discuss, offline.